tools/github-actions/release/action.yml

```diff
@@ -7,9 +7,17 @@ inputs:
   target:
     description: 'The branch to target'
     required: true
+  previousVersionsLimit:
+    description: 'Limit of previous versions to consider for the release notes'
+    required: false
+    default: '100'
```

Comment:
I think we are missing some details in the documentation of this GH Action

Reply:
There is not details regarding the parameters in the documentation, there is a link referring this file to get the list.
Or did you mean the description is unclear? If so can you suggest one?

Reply:
Since it is a particular behaviour for `.0` releases, I was thinking about adding some words in [the readme of the action](https://github.com/AmadeusITGroup/otter/blob/main/tools/github-actions/release/readme.md) explaining how the release notes is computed in case of `.0` releases.